Makefile

```diff
@@ -43,5 +46,5 @@ endif
 # Requires the ALCHEMY_API_KEY env var to be set.
 # The first argument determines the network (mainnet / rinkeby / ropsten / kovan / goerli)
 define network
-	https://eth-$1.alchemyapi.io/v2/${ALCHEMY_API_KEY}
+https://eth-$1.alchemyapi.io/v2/${ALCHEMY_API_KEY}
```

Comment:
had to remove the whitespace to make the command above work. Otherwise, we got something like `ETH_RPC_URL=    https://...` when calling `$(call network,$(network_name))`.